Formula/synergy-core.rb

```diff
@@ -0,0 +1,33 @@
+class SynergyCore < Formula
+  desc "Open source core of Synergy, the keyboard and mouse sharing tool"
+  homepage "https://symless.com/synergy"
+  url "https://github.com/symless/synergy-core.git",
+      :revision => "62ab8ffc4fdbc84b51552d600a8e11f7a7b78d84"
+  version "1.9.0-rc4"
```

Comment:
This would need to be for the latest stable release, which is currently 1.8.8 https://github.com/symless/synergy-core/releases/latest